### PR TITLE
Update index.rst

### PR DESCRIPTION
**Pull request by rmcabezon**
Just trying to add something

### CODE REVIEW

**Review by sekelle — APPROVED** [review]

**Comment by sekelle** [comment]
@jgphpc Could you check why this fails docvalidation?

File: docs/source/index.rst [hunk 1]
```diff
@@ -1,8 +1,11 @@
 ==================
 Welcome to SPH-EXA
 ==================
+SPH-EXA is a C++20 simulation code for hydrodynamics simulations (with gravity and other physics), parallelized with MPI, OpenMP, CUDA, and HIP.
```
Comment (jgphpc) [inline docs/source/index.rst]:
```suggestion
SPH-EXA is a C++20 simulation code for hydrodynamics simulations (with gravity
and other physics), parallelized with MPI, OpenMP, CUDA, and HIP.
```

File: docs/source/index.rst [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
 
-SPH-EXA is ...
+SPH-EXA is built with high performance, scalability, portability, and resilience in mind. Its SPH implementation is based on SPHYNX, ChaNGa, and SPH-flow, three SPH codes selected in the PASC SPH-EXA project to act as parent and reference codes to SPH-EXA.
```
Comment (jgphpc) [inline docs/source/index.rst]:
```suggestion
SPH-EXA is built with high performance, scalability, portability, and
resilience in mind. Its SPH implementation is based on SPHYNX, ChaNGa, and
SPH-flow, three SPH codes selected in the PASC SPH-EXA project to act as parent
and reference codes to SPH-EXA.
```

File: docs/source/index.rst [hunk 3]
```diff
@@ -1,8 +1,11 @@
...
+
+The performance of standard codes is negatively impacted by factors such as imbalanced multi-scale physics, individual time-stepping, halos exchange, and long-range forces. Therefore, the goal is to extrapolate common basic SPH features, and consolidate them in a fully optimized, Exascale-ready, MPI+X, SPH code: SPH-EXA.
```
Comment (jgphpc) [inline docs/source/index.rst]:
```suggestion
The performance of standard codes is negatively impacted by factors such as
imbalanced multi-scale physics, individual time-stepping, halos exchange, and
long-range forces. Therefore, the goal is to extrapolate common basic SPH
features, and consolidate them in a fully optimized, Exascale-ready, MPI+X, SPH
code: SPH-EXA.
```

**Review by jgphpc — APPROVED** [review]
just a formatting issue

**Pull request merged**